<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Right_Breast</class>
 <widget class="QDialog" name="Right_Breast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>10</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Result :</string>
    </property>
   </widget>
   <widget class="QLabel" name="DecisionVar_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(255, 0, 0);
border:0px;
}
</string>
    </property>
    <property name="text">
     <string>Cancer â˜ </string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>330</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageVAR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>18</y>
      <width>331</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:#4b6272;
border:0px;
}
QFrame{
	background-color: rgb(255, 255, 255);

border:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>330</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="Save">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>234</width>
        <height>50</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	text-align:left;
	color:#fff;
	border:0px;
	border-radius:20px;
	font-size:15px;
	cursor: pointer;
	background-color:#538fff;
}
QPushButton:hover{
background-color:#3668ff;
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="icon">
       <iconset resource="file.qrc">
        <normaloff>assets/img/FAB-1.png</normaloff>assets/img/FAB-1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Next">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>234</width>
        <height>50</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	text-align:left;
	color:#fff;
	border:0px;
	border-radius:20px;
	font-size:15px;
	cursor: pointer;
	background-color:#538fff;
}
QPushButton:hover{
background-color:#3668ff;
}</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
      <property name="icon">
       <iconset resource="file.qrc">
        <normaloff>assets/img/path395.png</normaloff>assets/img/path395.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
